src/app/hooks/useGa.ts

```diff
@@ -51,9 +51,7 @@ export const useGa = (): {
     record: (options: Ga4EventOptions) => void;
   };
 } => {
-  const debugMode =
-    process.env.NEXT_PUBLIC_NODE_ENV !== "production" &&
-    process.env.NODE_ENV !== "test";
+  const debugMode = process.env.NEXT_PUBLIC_GA4_DEBUG_MODE !== "true";
```

Comment:
I'm confused, should this be `!==` or `===`?
Seems odd that it's `debugMode` where `_GA4_DEBUG_MODE !== true` since those sound conflicting.

Reply:
🤦 thanks